src/azul/strings.py

```diff
@@ -48,20 +55,53 @@ def pluralize(word: str, count: int) -> str:
     return result
 
 
-def unwrap(string: str):
+def format_description(string: str) -> str:
     """
-    Remove newlines and leading/trailing whitespace, and consolidate
-    newline-adjacent whitespace.
+    Remove common leading whitespace from every line in text.
     Useful for processing triple-quote strings.
 
-    >>> unwrap(' ce nest pas \\n une chaine \\n de plusieurs lignes. ')
-    'ce nest pas une chaine de plusieurs lignes.'
+    :param string: The string to unwrap
+
+    >>> format_description(" c'est \\n une chaine \\n de plusieurs lignes. ")
+    "c'est \\nune chaine \\nde plusieurs lignes. "
 
-    >>> unwrap('''
+    >>> format_description('''
     ...        Multi-lined,
     ...        indented,
     ...        triple-quoted string.
     ...        ''')
-    'Multi-lined, indented, triple-quoted string.'
+    '\\nMulti-lined,\\nindented,\\ntriple-quoted string.\\n'
+    """
+    return dedent(string)
+
+
+def format_description_key(kwargs: MutableMapping[str, Any]) -> None:
```

Comment:
And this one is even more specific to OpenAPI spec, so it's definitely in the wrong place here.